.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  - push
+
+jobs:
+  all:
+    name: all
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+          registry-url: https://registry.npmjs.org
+
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/yarn
+          key: v1-${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: v1-${{ runner.os }}-yarn-
+
+      - run: yarn install --frozen-lockfile
```

Comment:
when I ran this locally it claimed `--frozen-lockfile` was deprecated in favor of `--immutable` 🤷

Reply:
sounds like you were using yarn 2?